rskj-core/src/test/java/co/rsk/peg/BridgeSupportFlyoverTest.java

```diff
@@ -1446,61 +1451,59 @@ public void registerFastBridgeBtcTransaction_funds_sent_to_retiring_fed_after_RS
     }
 
     @Test
-    public void registerFastBridgeBtcTransaction_funds_sent_to_retiring_fed_surpasses_locking_cap_after_RSKIP293() throws BlockStoreException, BridgeIllegalArgumentException, IOException {
-        // Refund should be transfer to the user
+    public void registerFastBridgeBtcTransaction_funds_sent_to_retiring_fed_surpasses_locking_cap_should_refund_user_before_RSKIP293() throws BlockStoreException, BridgeIllegalArgumentException, IOException {
         BigInteger result = sendFundsSurpassesLockingCapToAnyAddress(
             (bridgeConstants, activeFederationAddress, retiringFederationAddress) -> {
-                Coin valueToSend = Coin.COIN;
                 BtcTransaction tx = new BtcTransaction(bridgeConstants.getBtcParams());
-                tx.addOutput(valueToSend, retiringFederationAddress);
+                tx.addOutput(bridgeConstantsRegtest.getMaxRbtc(), retiringFederationAddress);
                 return tx;
             },
             false,
-            bridgeConstantsRegtest.getMaxRbtc()
+            false
         );
-        Assert.assertEquals(FastBridgeTxResponseCodes.REFUNDED_USER_ERROR.value(), result.longValue());
+        Assert.assertEquals(FastBridgeTxResponseCodes.UNPROCESSABLE_TX_VALUE_ZERO_ERROR.value(), result.longValue());
```

Comment:
You get zero value because there are no funds sent to the active federation. It's never reaching the check for locking cap that it's what we actually want to test, isn't it?